fs/config/rc_test.go

```diff
@@ -68,6 +69,7 @@ func TestRc(t *testing.T) {
 	})
 
 	t.Run("ListRemotes", func(t *testing.T) {
+		_ = os.Setenv("RCLONE_CONFIG_MY-LOCAL_TYPE", "local")
```

Comment:
I would tidy this env var up at the end of the function otherwise it might interfere with other tests.

I'd also check the error from os.Setenv which is very easy to do in the test functions.

Something like this (untested)


```go
assert.NoError(t, os.Setenv("RCLONE_CONFIG_MY-LOCAL_TYPE", "local"))
defer func() {
    assert.NoError(t, os.Unsetenv("RCLONE_CONFIG_MY-LOCAL_TYPE"))
}()
```

Reply:
Yes this is good practice - sorry for not thinking about it. Done.